aea/test_tools/test_cases.py

```diff
@@ -208,7 +233,7 @@ def run_agent(self, *args: str) -> subprocess.Popen:
 
     def terminate_agents(
         self,
-        subprocesses: Optional[List[subprocess.Popen]] = None,
+        *subprocesses: subprocess.Popen,
```

Comment:
I think it should be:
```suggestion
        *subprocesses: Optional[subprocess.Popen] = None,
```

Reply:
InvalidSyntax. We're unable to assign a default param to *args.

Reply:
Thanks